integration/src/with-webhooks/handlers/products-update.handler.ts

```diff
@@ -4,7 +4,7 @@ import {
 } from '@rh-nestjs-shopify/webhooks';
 import { Inject, Scope } from '@nestjs/common';
 import { REQUEST } from '@nestjs/core';
-import { IncomingMessage } from 'http';
+import { IncomingMessage } from 'node:http';
```

Comment:
What is difference between `http` and `node:http`?

Reply:
[link](https://nodejs.org/api/modules.html#core-modules)
cho chắc chắn nó luôn trả về thư viện của nodejs module